client/app/src/pages/admin/users/user-editor/user-editor.component.html

```diff
@@ -53,6 +54,11 @@
                     <input class="form-control" name="name" [(ngModel)]="user.name" type="text" required />
                     <div class="text-danger" *ngIf="editUser.form.get('name')?.errors?.['required']">This field is mandatory </div>
                 </div>
+                <div class="form-group">
+                    <label>{{ 'Fiscal Code' | translate }}</label>
+                    <input class="form-control" name="fiscalcode" [(ngModel)]="user.fiscal_code" type="text" required />
```

Comment:
In the context of GlobaLeaks few aspects need to be changed to be integrated
- "Fiscal Code" should be renamed "IDP-ID" because not every country have a "Fiscal Code"  while the functionality can be abstracted to be "Identity Povider ID" that is generic enough to satisfy any integration with external IDPs.
- The field should not be set as mandatory or it will be impossible to have users that do not use an identity provider or even anonymous like in some of the use cases supported by the application.

Reply:
![image](https://github.com/user-attachments/assets/1f412b73-bfdb-43ba-9bde-672e58167f5b)

Reply:
![Screenshot from 2024-11-15 19-06-28](https://github.com/user-attachments/assets/26389947-fdac-4485-a2d6-9834c99231a6)